.markdownlint.jsonc

```diff
@@ -30,8 +30,12 @@
       "script"
     ]
   },
+  // This warns if emails are used without '<' and '>'. That's ok.
```

Comment:
```suggestion
  // This warns if URLs or emails are used without '<' and '>'. That's ok.
```

Reply:
Is it ok? Can't we fix problematic links in the README instead?

Reply:
Sure, let me fix the emails in README then